### PR TITLE
Split bffnt into bffnt.py and bcfnt.py

### PR DESCRIPTION
**Pull request by Tyulis**
My previous PR was a bit... bugged because of unexpected differences between CFNT v3 and FFNT v3. Now, these two formats are handled by two separated programs (globally sames, only some headers changed, but that was difficult to merge into a single program). This will fix errors with BFFNT v3. Only the ETC1 decompression error still stays.

### CODE REVIEW

**Comment by Tyulis** [comment]
Don't know why there are these conflicts, but this is normal

File: bffnt.py [hunk 1]
```diff
@@ -503,32 +522,38 @@ def save(self, filename):
...
+
+
+        if self.version not in VERSIONS:
+            print('Unknown version: 0x%08x (expected 0x%08x or 0x%08x)' % (self.version, 0x03000000, 0x04000000))
             self.invalid = True
             return
 
         if header_size != FFNT_HEADER_SIZE:
             print('Invalid header size: %d (expected %d)' % (header_size, FFNT_HEADER_SIZE))
+            blinh
```
Comment (ObsidianX) [inline bffnt.py]: Conflict may be because you were developing on an old branch that we already merged into `master`

File: bffnt.py [hunk 2]
```diff
@@ -255,22 +257,21 @@ def load(self, json_filename):
...
 
-    def _int_sort(self, x, y):
-        x = int(x, 10)
-        y = int(y, 10)
-        if x < y:
-            return -1
-        if x > y:
-            return 1
-        return 0
+    def _int_sort(self, n):
```
Comment (ObsidianX) [inline bffnt.py]: This isn't a sorting function anymore, just a conversion from string to base 10.

File: bcfnt.py [hunk 3]
```diff
@@ -0,0 +1,1235 @@
...
+
+
+        if self.version not in VERSIONS:
+            print('Unknown version: 0x%08x (expected 0x%08x or 0x%08x)' % (self.version, 0x03000000, 0x04000000))
+            self.invalid = True
+            return
+
+        if header_size != CFNT_HEADER_SIZE:
+            print('Invalid header size: %d (expected %d)' % (header_size, CFNT_HEADER_SIZE))
+            blinh
```
Comment (ObsidianX) [inline bcfnt.py]: Bad line

File: bcfnt.py [hunk 4]
```diff
@@ -0,0 +1,1235 @@
...
+            try:  #Python2
+                utf16 = unichr(entry)
+            except:  #Python3
+                utf16 = chr(entry)
+            if utf16 in glyph_map:
+                cmap['entries'][utf16] = glyph_map[utf16]
+
+        self.cmap_sections = [cmap]
+
+    def _int_sort(self, n):
```
Comment (ObsidianX) [inline bcfnt.py]: Same comment here: this isn't a sorting function anymore. What's the purpose?

**Comment by Tyulis** [comment]
I tested, and saw that sorting functions are not anymore real sorting functions, but just a conversion to then sort with it. I tested with python2 and 3.

**Comment by ObsidianX** [comment]
Gotta clean up the branch before I can merge

**Comment by ObsidianX** [comment]
Just a heads up this branch is still conflicting and needs to be cleaned up before I can accept it.

**Comment by Tyulis** [comment]
That's because I didn't updated the fork before editing. Sorry, I'll open another pull request with a cleaner fork.

**Pull request closed without merging**